packet_test.go

```diff
@@ -85,4 +85,32 @@ func TestPacket(t *testing.T) {
 	b = []byte{}
 	require.NoError(t, pkt6.FromData(b))
 	require.Equal(t, b, pkt6.Data())
+
+	t.Run("CopyProperties", func(t *testing.T) {
+		pktCopy := AllocPacket()
+		require.NotNil(t, pktCopy)
+		defer pktCopy.Free()
+
+		require.NotEqual(t, pkt2.c.dts, pktCopy.c.dts)
```

Comment:
Can you use `.Dts()` instead of `.c.dts` everywhere below?

Reply:
Fixed.